### PR TITLE
[WIP] Updating provider summary tables

### PR DESCRIPTION
**Pull request by tkhamis**
{{ pytest :  cfme/tests/containers/test_reload_button_provider.py --use-provider  ocp-36-hawk }}
 Updating automation after provider summary tables fields were changed  .

### CODE REVIEW

**Comment by tkhamis** [comment]
This issue is blocked by WRAPNAPI 3.0

File: cfme/containers/provider/__init__.py [hunk 1]
```diff
@@ -230,7 +230,11 @@ def num_project(self):
     @num_project.variant('ui')
     def num_project_ui(self):
         view = navigate_to(self, "Details")
-        return int(view.entities.summary("Relationships").get_text_of("Projects"))
+        if self.appliance.version < "5.9":
```
Comment (jawatts) [inline cfme/containers/provider/__init__.py]: Please use the Widgetastic VersionPick class here:
https://github.com/jawatts/widgetastic.core/blob/master/src/widgetastic/utils.py#L241

Example:
```
VersionPick({
        Version.lowest(): 'Projects',
        '5.9': 'Container Projects'
    })
```

**Comment by dajoRH** [comment]
Lint report for commit 0ddd869e5aba0c326cbd59a39194d4490f534597:

Lint OK! :cake: :punch: :cookie: .

* 8f32e8c72b50de1eaa4773d3b254dc91eebfe599
  * Merge commit detected!!! :bangbang:

**The PR might not be acceptable in this state** :rage4:.
There is a merge commit in the branch which we do not like in PRs so please, get rid of it.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 956e824511be90e0ca89f9b1c31acaf99d489a0f:

Lint OK! :cake: :punch: :cookie: .

* 8f32e8c72b50de1eaa4773d3b254dc91eebfe599
  * Merge commit detected!!! :bangbang:

**The PR might not be acceptable in this state** :rage4:.
There is a merge commit in the branch which we do not like in PRs so please, get rid of it.

*CFME QE Bot*

**Pull request closed without merging**